Epsilon.Host.Frontend/src/views/AuthorizeButton.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <button @click="openWindow">Click here to authorize</button>
+</template>
+
+<script lang="ts" setup>
+import { ref } from "vue"
+import router from "../router"
+
+const windowRef = ref()
+
+function openWindow(): void {
+    windowRef.value = window.open(
+        "https://localhost:7084/auth/challenge",
+        "",
+        "width=600,height=400"
+    )
+
+    const timer = setInterval(() => {
+        if (windowRef.value.closed) {
+            router.push("/")
+            clearInterval(timer)
+        }
+    }, 500)
+}
+</script>
+
+<style scoped>
+button {
+    background: #535bf2;
+}
+</style>
```

Comment:
Why is this in this PR @NealGeilen

Reply:
Part of resolved merge conflicts and GIT mess